install-vdvs.ps1

```diff
@@ -34,7 +34,8 @@
 # Command line parameters
 param (
     [string] $uri,
-    [switch] $uninstall
+    [switch] $uninstall,
+    [switch] $Force
```

Comment:
Since "-Force" option can be used by both installation and uninstallation, it's better to document this somewhere, either in the description (to explain the meaning of this option in installation and uninstallation) and/or in the examples (by adding 2 more examples).